_docker/lib/export/export.rb

```diff
@@ -42,6 +43,7 @@ def export!
 
     if !@rsync_destination.nil? and !@rsync_destination.empty?
       @rsync_handler.rsync_static_export(export_directory, @rsync_destination, true)
+      @export_diff.modified_content_urls(export_directory)
```

Comment:
I'm also wondering if we need to do a full export if we already know the diffs. Can we simply feed httrack a list of URLs that are we need to export after we've diffed the sitemap.xml files?

Reply:
@LightGuard The honest answer is "I don't know". 

I'll pull that idea out as another potential improvement to the export process as it feels beyond the scope of this story and it would involve further re-working of the export code.

I've created [DEVELOPER-5381](https://issues.jboss.org/browse/DEVELOPER-5381) to spike that issue.

cheers,

Rob